src/main/java/athena/TimeAllocate.java

```diff
@@ -0,0 +1,57 @@
+package athena;
+
+import athena.task.Task;
+import athena.task.taskfilter.FlexibleTimeFilter;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+
+public class TimeAllocate {
+    ArrayList<Task> flexibleTask;
+    ArrayList<Task> fixedTask;
```

Comment:
Can we use `TaskList` instead? So that we can make use of methods from `TaskList` if needed.

Reply:
I need the sorting according to time so ArrayList is actually a bit better for this scenario, I might need a bit of help with the time sorting, not very clear how to do it